frontend/src/features/mapSideBar/tabs/propertyDetails/update/UpdatePropertyDetailsForm.tsx

```diff
@@ -0,0 +1,267 @@
+import { Button } from 'components/common/buttons';
+import { Input, Multiselect, Select, Text, TextArea } from 'components/common/form';
+import { RadioGroup } from 'components/common/form/RadioGroup';
+import { YesNoSelect } from 'components/common/form/YesNoSelect';
+import { Scrollable } from 'components/common/Scrollable/Scrollable';
+import * as API from 'constants/API';
+import { PropertyAdjacentLandTypes, PropertyTenureTypes } from 'constants/index';
+import { Form, FormikProps, getIn } from 'formik';
+import { useLookupCodeHelpers } from 'hooks/useLookupCodeHelpers';
+import React, { useEffect } from 'react';
+import { ButtonToolbar, Col, Row } from 'react-bootstrap';
+import styled from 'styled-components';
+import { stringToBoolean } from 'utils/formUtils';
+
+import { Section } from '../../Section';
+import { SectionField, StyledFieldLabel } from '../../SectionField';
+import { InlineContainer, LeftBorderCol } from '../../SectionStyles';
+import { LandMeasurementTable } from './components/LandMeasurementTable';
+import { VolumetricMeasurementTable } from './components/VolumetricMeasurementTable';
+import {
+  PropertyAdjacentLandFormModel,
+  PropertyAnomalyFormModel,
+  PropertyRoadFormModel,
+  PropertyTenureFormModel,
+  UpdatePropertyDetailsFormModel,
+} from './models';
+
+export interface IUpdatePropertyDetailsFormProps
+  extends FormikProps<UpdatePropertyDetailsFormModel> {
+  onCancel: () => void;
+}
+
+export const UpdatePropertyDetailsForm: React.FC<IUpdatePropertyDetailsFormProps> = ({
+  values,
+  setFieldValue,
+  onCancel,
+}) => {
+  // Lookup codes
+  const { getByType, getOptionsByType } = useLookupCodeHelpers();
+  const volumetricTypeOptions = getOptionsByType(API.PROPERTY_VOLUMETRIC_TYPES);
+  const propertyTypeOptions = getOptionsByType(API.PROPERTY_LAND_PARCEL_TYPES);
+  const anomalyOptions = getByType(API.PROPERTY_ANOMALY_TYPES).map(x =>
+    PropertyAnomalyFormModel.fromLookup(x),
+  );
+  const tenureOptions = getByType(API.PROPERTY_TENURE_TYPES).map(x =>
+    PropertyTenureFormModel.fromLookup(x),
+  );
+  const roadTypeOptions = getByType(API.PROPERTY_ROAD_TYPES).map(x =>
+    PropertyRoadFormModel.fromLookup(x),
+  );
+  const adjacentLandOptions = getByType(API.PROPERTY_ADJACENT_LAND_TYPES).map(x =>
+    PropertyAdjacentLandFormModel.fromLookup(x),
+  );
+
+  // multi-selects
+  const tenureStatus = getIn(values, 'tenures') as PropertyTenureFormModel[];
```

Comment:
Why use getIn instead of just values.tenures ? values.tenures seems better because it is more concise and doesn't rely on type coercion.

Reply:
no hard preference there, can change. was kinda using formik helpers but agree on type coercion